bin/convert-argv.js

```diff
@@ -86,7 +86,9 @@ module.exports = function(yargs, argv, convertOptions) {
 							registerCompiler(moduleDescriptor[i]);
 							break;
 						} catch(e) {
-							// do nothing
+							if(i === moduleDescriptor.length - 1 && e.message.match(/Cannot find module/)) {
+								throw new Error("Cannot resolve config file. Install appropriate compiler and try again.");
```

Comment:
Add a note about which missing package name.

It would be great to include the old errors message and stack.

``` js
catch(e) {
  e.message = `Cannot ... ` + e.message;
  throw e;
}
```